sklearn/metrics/classification.py

```diff
@@ -1597,36 +1602,53 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None):
     The logarithm used is the natural logarithm (base-e).
     """
     lb = LabelBinarizer()
-    T = lb.fit_transform(y_true)
```

Comment:
this block was moved to line 1620 in the diff. the variable `T` was renamed to `transformed_labels` and the variable `Y` was renamed to `y_pred`